lib/scss_lint/linter/space_around_operator.rb

```diff
@@ -0,0 +1,57 @@
+module SCSSLint
+  # Checks for space around operators on values.
+  class Linter::SpaceAroundOperator < Linter
+    include LinterRegistry
+
+    def visit_script_operation(node)
+      source = source_from_range(node.source_range)
```

Comment:
I propose changing to:

```ruby
source = source_from_range(node.source_range).gsub(/\)\z/, '')
```

to remove trailing parentheses from the lint message.